indigo_content_api/v2/serializers.py

```diff
@@ -68,7 +68,8 @@ class PublishedDocumentSerializer(DocumentSerializer, PublishedDocUrlMixin):
     points_in_time = serializers.SerializerMethodField()
     publication_document = serializers.SerializerMethodField()
     taxonomies = serializers.SerializerMethodField()
-    as_at_date = serializers.DateField(source='work.as_at_date')
+    # TODO: double check this
+    as_at_date = serializers.DateField(source='work.computed_as_at_date')
```

Comment:
You don't need to worry about this if you don't change the existing `as_at_date` API on the work object.